cedar-drt/fuzz/src/lib.rs

```diff
@@ -184,13 +187,46 @@ pub fn run_val_test(
     let validator = Validator::new(schema.clone());
     let (rust_res, rust_validation_dur) = time_function(|| validator.validate(policies, mode));
     info!("{}{}", RUST_VALIDATION_MSG, rust_validation_dur.as_nanos());
-
     let definitional_res = custom_impl.validate(&schema, policies, mode);
+    compare_validation_results(
+        policies,
+        &schema,
+        custom_impl.validation_comparison_mode(),
+        rust_res,
+        definitional_res,
+    );
+}
+
+pub fn run_level_val_test(
+    custom_impl: &impl CedarTestImplementation,
+    schema: ValidatorSchema,
+    policies: &ast::PolicySet,
+    mode: ValidationMode,
+    level: i32,
+) {
+    let validator = Validator::new(schema.clone());
+    let (rust_res, rust_validation_dur) =
+        time_function(|| validator.validate_with_level(policies, mode, level as u32));
+    info!("{}{}", RUST_VALIDATION_MSG, rust_validation_dur.as_nanos());
+    let definitional_res = custom_impl.validate_with_level(&schema, policies, mode, level);
+    compare_validation_results(
+        policies,
+        &schema,
+        custom_impl.validation_comparison_mode(),
+        rust_res,
+        definitional_res,
+    );
+}
 
+fn compare_validation_results(
+    policies: &ast::PolicySet,
+    schema: &ValidatorSchema,
+    comparison_mode: ValidationComparisonMode,
+    rust_res: ValidationResult,
+    definitional_res: TestResult<TestValidationResult>,
+) {
     match definitional_res {
         TestResult::Failure(err) => {
-            // TODO(#175): For now, ignore cases where the Lean code returned an error due to
-            // an unknown extension function.
             if !err.contains("unknown extension function")
                 && !err.contains("unknown extension type")
             {
```

Comment:
intentionally removing this comment? Looks like the associated code still exists